_component_design/toast.md

```diff
@@ -5,40 +5,136 @@ layout: component
+
+### Timing
+
+The toast fades into place over 400ms, sits in place for eight seconds, and fades out over 400ms. If the user happens to hover over the toast while it is fading out, it reanimates immediately (0.0ms) back to full opacity. Likewise, if the user is hovering over the toast at any point, it persists. When the mouse is removed from the expired toast, the toast persists for three more seconds then fades over the normal 400ms. 
+
+{% endcolumn %}
+
+</div>
+
+## Variations
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Without a CTA (call to action)
+
+Clicking on the “X” dismisses the toast notification. There are no other actions that can be taken on this type of toast notification.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Toast notifications without CTA (call to action)"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/notifications/toasts/toast-wo-cta.svg)
 {% endfigure %}
 {% endcolumn %}
+
 </div>
+
 <div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Action Buttons"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/toast-alert-action-buttons.svg){:width="100%"}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### With a CTA
+
+There should only be one CTA per toast.
+
+In the Informational toast notification example here, the height of the toast increases to 96px to maintain the 8px spacing between the text and the CTA, and the 12px spacing between the CTA and the bottom of the toast. (The width always stays consistent at 350p
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Toast notifications with CTA (call to action)"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/notifications/toasts/toast-w-cta.svg)
 {% endfigure %}
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Error"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/toast-error.svg){:width="100%"}
+
+</div>
+
+## Behaviors
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Placement, animations, and stacking
+
+Toasts appear in the top right corner, 24px underneath the navigation bar. If more than one toast notifications are triggered at one time, they can stack (24px apart from each other), with newer toasts coming in at the top.
+
+The toast fades into place over 400ms, sits in place for 8 seconds, and fades out over 400ms. If the user happens to hover over the toast while it is fading out, it immediately fades back to full transparency. Likewise, if the user is hovering over the toast at any point, it persists. When the mouse is removed from the expired toast, it fades after 3 seconds. 
```

Comment:
Duplicate paragraph. Just link to the timing section instead or move the timing section here.

Reply:
s/happens to hover/hovers
s/is hovering over/hovers over

Reply:
Moved Timing section to this location and deleted from Default layout area. Seems to make more sense here.